ddapm_test_agent/agent.py

```diff
@@ -387,6 +388,60 @@ async def _tracestats_by_session(self, token: Optional[str]) -> List[v06StatsPay
                 stats.append(s)
         return stats
 
+    async def _integration_requests_by_session(
+        self, token: Optional[str], include_sent_integrations: Optional[bool] = False
+    ) -> List[Dict]:
+        """Get all requests with an associated tested Integration."""
+        integration_requests: List[Tuple[Request, Dict]] = []
+        for req in self._requests_by_session(token):
+            # see if the request was to update with a newly tested integration
+            if req.match_info.handler == self.handle_put_tested_integrations:
+                if "integration" not in req:
+                    data = json.loads(await req.read())
+                    integration_name = data.get("integration_name", None)
+                    integration_version = data.get("integration_version", None)
+                    req["integration"] = Integration(
+                        integration_name=integration_name,
+                        integration_version=integration_version,
+                        dependency_name=data.get("dependency_name", integration_name),
+                        version_sent=f"{integration_name}@{integration_version}" in ",".join(os.listdir("artifacts")),
+                    )
+                    req["tracer_version"] = data.get("tracer_version", None)
+                    req["tracer_language"] = data.get("tracer_language", None)
+                    integration_requests.append(req)
+                elif include_sent_integrations:
+                    integration_requests.append(req)
+            # check if integration data was provided in the trace request instead
+            elif (
+                "_dd_trace_env_variables" in req
+                and "DD_PLUGIN" in req["_dd_trace_env_variables"]
```

Comment:
since we're referring to them as "integrations" in the test agent, does it make sense to use `DD_INTEGRATION` as the env var?